src/dotty/tools/dotc/typer/EtaExpansion.scala

```diff
@@ -48,13 +48,11 @@ object EtaExpansion {
   }
 
   /** Lift a function argument, stripping any NamedArg wrapper */
```

Comment:
I think the documentation needs to be updated, since we don't strip the NamedArg anymore.